hack/update-codegen.sh

```diff
@@ -28,7 +28,35 @@ fi
 
 source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/library.sh
 
-# Compute _example checksum for all configmaps.
+boilerplate="${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt"
+
+# Parse flags to determine if we should generate protobufs.
+generate_protobufs=0
+while [[ $# -ne 0 ]]; do
+  parameter=$1
+  case ${parameter} in
+    --generate-protobufs) generate_protobufs=1 ;;
+    *) abort "unknown option ${parameter}" ;;
+  esac
+  shift
+done
+readonly generate_protobufs
+
+if (( generate_protobufs )); then
+  echo "Generating protocol buffer code"
+  protos=$(find . -name '*.proto' | grep -v vendor | grep -v third_party)
+  for proto in $protos
+  do
+    protoc "${REPO_ROOT_DIR}/$proto" -I="${REPO_ROOT_DIR}" --gogofaster_out=plugins=grpc:.
```

Comment:
this will work when everything is in one file, but in general we should probably do basedir on the found file and feed all proto files together.
E.g. standard policy is to have data types in one proto file and the service definition in the other.

Reply:
Not sure why that matters for protoc generation :thinking:. Do I need to load the two files because one depends on each other? I'd have to try that out.

Reply:
I think you need the list.

Reply:
I tried it, it doesn't seem to be the case. You just import the file with the messages and you're good to go it seems. As such, I'll leave this as-is for now.